tests/foreman/api/test_rhcloud_inventory.py

```diff
@@ -108,7 +108,8 @@ def test_rhcloud_inventory_api_e2e(
     infrastructure_type = [
         host['system_profile']['infrastructure_type'] for host in json_data['hosts']
     ]
-    assert 'physical' and 'virtual' in infrastructure_type
+    assert 'physical'
+    assert 'virtual' in infrastructure_type
```

Comment:
Same issue here as above

Reply:
same answer as the previous two :)
if we agree this is a test logic issue, I can fix them to be the intended behavior.

Reply:
Its a test logic issue for sure!

Reply:
@JacobCallahan @ogajduse it's test logic, as @jyejare mentioned. both the `virtual` and `physical` should be present in `infrastructure_type`.